packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.component.tsx

```diff
@@ -173,9 +198,26 @@ const VisitSummary: React.FC<VisitSummaryProps> = ({ visit, patientUuid }) => {
           <TabPanel>
             <MedicationSummary medications={medications} />
           </TabPanel>
-          <TabPanel>
-            <VisitsTable visits={mapEncounters(visit)} showAllEncounters={false} patientUuid={patientUuid} />
-          </TabPanel>
+          {showFilledFormsInTabs ? (
+            visit?.encounters?.length > 0 && foundEncounter ? (
+              <TabPanel key={selectedIndex}>
+                <OHRIForm
+                  patientUUID={patientUuid}
+                  formUUID={foundEncounter.form?.uuid}
+                  encounterUUID={foundEncounter.uuid}
+                  mode="view"
+                />
+              </TabPanel>
+            ) : (
+              <TabPanel key={selectedIndex}>
+                <></>
+              </TabPanel>
+            )
```

Comment:
This should also render all the summary for all the forms, since we are rendering all the forms as tabs, the Carbon maps the tab index to tabPanel's index.

Reply:
it will render a specific form based on the tab selected